code-compass.el

```diff
@@ -186,10 +192,10 @@
   repository)
 
 (defun c/produce-cloc-report (repository)
-  "Create cloc report for REPOSITORY."
+  "Create cloc report for REPOSITORY. To filter specific subdirectories out of this report, edit the variable `c/exclude-directories'."
   (message "Producing cloc report...")
   (shell-command
-   (format "(cd %s; cloc ./ --by-file --csv --quiet) > cloc.csv" repository))
+   (format "(cd %s; cloc ./ --by-file --csv --quiet --skip-uniqueness --exclude-dir=%s) > cloc.csv" repository (string-join c/exclude-directories ",")))
```

Comment:
Looking this over, I should perhaps remove the `--skip-uniqueness` flag; it was a performance optimization I made while experimenting, but probably users should add nonstandard flags themselves.